core/src/lib.rs

```diff
@@ -41,6 +41,7 @@
     trivial_numeric_casts,
     unused_extern_crates
 )]
+#![feature(cell_update)]
```

Comment:
Is using this feature really worth it? While the code looks prettier indeed I think we can define `Cell::update` locally, right?

Reply:
will remove it